virttest/utils_misc.py

```diff
@@ -851,6 +851,44 @@ def archive_as_tarball(source_dir, dest_dir, tarball_name=None,
     tarball.close()
 
 
+def get_guest_service_status(session, service, service_former=None):
+    """
+    Get service's status in guest.
+    Return 'active' for 'running' and 'active' case.
+    Return 'inactive' for 'stopped' and 'inactive' case.
+
+    :param session: An Expect or ShellSession instance to operate on
+    :type devices: Expect or ShellSession class
+    :param service: Service name that we want to check status.
+    :type service: String
+    :param service_former: Service's former name
+    :type service: String
+    :return: 'active' or 'inactive'
+    :rtype: String
+    """
+    output = ""
+    cmd = ("service %(key)s status || "
+           "systemctl %(key)s.service status || "
+           "status %(key)s")
+    try:
+        output = session.cmd_output(cmd % {'key': service})
+    except Exception:
+        if not service_former:
```

Comment:
If not service_former, that mean service_former should be a false values, view from your code, it should be None or "",  so that next line should be execute without any replacement. so what you want to get?